frontend/src/embedding-sdk-bundle/components/public/dashboard/EditableDashboard/EditableDashboard.stories.tsx

```diff
@@ -32,12 +32,7 @@ const meta = {
 
     "dataPickerProps.entityTypes": {
       control: "check",
-      options: [
-        "model",
-        "table",
-        "question",
-      ] satisfies SdkQuestionProps["entityTypes"],
-      description: "`question` doesn't have effect on simple data picker",
+      options: ["model", "table"] satisfies SdkQuestionProps["entityTypes"],
```

Comment:
this is the storybook, I suspect I added question just to satisfy the type as you can see in he description. Removing this should be no problem, as we don't want users to know we can pass it, I mean it's used in the edge case where the combined tables + models is 100 or more.